texar/torch/run/executor.py

```diff
@@ -1147,8 +1147,16 @@ def save(self, path: Optional[str] = None,
             torch.save(train_state, str(ckpt_path))
         else:
             torch.save(self.model.state_dict(), str(ckpt_path))
+
```

Comment:
Why do we discard these changes? I think these lines of code are used to resolve the `meta-info` issue you mentioned.

Reply:
Sorry for the confusion, I later realized that we had to store metric objects in order to compare them, so I revised the commit, but forgot to edit the PR description.

Instead of only saving the metric values, I just changed the pickle behavior for `LR` so that it doesn't save the optimizer.